AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5285] SG2 | Create "Subcategory Featured Content" Organism

## Ticket(s)

**Jira Ticket**
- [EWL-5285: SG2 | Create "Subcategory Featured Content" Organism](https://issues.ama-assn.org/browse/EWL-5285)

## Description
As a user I can preview a piece of content and then click on it to learn more.

This update creates the twig template, descriptive markdown, and relevant scss required to make the subcategory featured content organism. Test content uses articles. The grid_column_layout mixin was refactored to use flexbox.

## To Test

- Pull `feature/EWL-5285-subcategory-featured-content-organism` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Organism > Subcategory Featured Content](http://localhost:3000/patterns/03-organisms-subcategory-featured-content/03-organisms-subcategory-featured-content.html).
- Confirm organism looks as shown in Zeplin mock up.
- Click on the gear icon in the top right corner of the screen and then click "show pattern info". Relevant information about the subcategory featured content should appear.

## Visual Regressions

Updated reference images for subcategory featured content and promo group backstop tests have been included in this PR. Promo group organism needed updating due to change in grid layout functionality. This does not affect the overall look of the error page which consumes promo group. 

I am experiencing backstop failures for `visual subcategory` and `visual subcategory exploration with images,` however it does not show any change between references.


## Relevant Screenshots/GIFs
![image](https://user-images.githubusercontent.com/4438120/41871425-8b27a3a4-7884-11e8-9fdb-85e2293d78dc.png)





## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
